dbms/src/DataStreams/CreatingSetsBlockInputStream.cpp

```diff
@@ -120,7 +120,7 @@ void CreatingSetsBlockInputStream::createAll()
             for (auto & elem : subqueries_for_sets)
             {
                 if (elem.second.join)
-                    elem.second.join->setBuildTableState(Join::BuildTableState::WAITING);
+                    elem.second.join->setInitActiveBuildConcurrency();
```

Comment:
Can we move to `PhysicalJoin`?

Reply:
Currently, it must be set after compile, see the comments in https://github.com/pingcap/tiflash/pull/6690/files#diff-e6f85ec76cd474ca862de277609c1dfbc11aa70692a3acf3f4e85fefb649cb65R473-R474